src/Yarhl/FileSystem/Node.cs

```diff
@@ -48,6 +49,32 @@ public Node(string name, IFormat initialFormat)
             ChangeFormat(initialFormat);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Node"/> class.
+        /// </summary>
+        /// <param name="node">The original node.</param>
+        /// <remarks><para>It makes a copy of the original node.
+        /// The original format is deep copied. See <see cref="ICloneableFormat"/> for details.</para></remarks>
+        public Node(Node node)
```

Comment:
Should we copy add or copy recursively the children?
If not, I think we should specify the behavior in the `remarks` of the doc.

Reply:
The children are already copied recursively because `NodeContainerFormat` implements `DeepClone()` and there is where the copies are made.
I'll add the comment in case the node has children but its format is not `NodeContainerFormat`.